entries/i/independentreserve.com.json

```
@@ -0,0 +1,43 @@
+{
+  "Independent Reserve": {
+    "domain": "independentreserve.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://blog.independentreserve.com/knowledge-base/setting-up-2fa-using-google-authenticator",
+    "keywords": [
+      "cryptocurrencies"
+    ],
+    "regions": [
+      "at",
+      "au",
+      "be",
+      "ca",
+      "ch",
+      "cz",
+      "de",
+      "dk",
+      "fi",
+      "fr",
+      "gb",
+      "hk",
+      "hu",
+      "is",
+      "it",
+      "jp",
+      "kr",
+      "li",
+      "lu",
+      "mc",
+      "mo",
+      "no",
+      "nz",
+      "pl",
+      "se",
+      "sg",
+      "si",
+      "sk",
+      "tw"
```

Missing regions according to https://www.independentreserve.com/help/getting-started: gl ie im ky nl pt 
```suggestion
      "at",
      "au",
      "be",
      "ca",
      "ch",
      "cz",
      "de",
      "dk",
      "fi",
      "fr",
      "gb",
      "gl",
      "hk",
      "hu",
      "ie",
      "im",
      "is",
      "it",
      "jp",
      "kr",
      "ky",
      "li",
      "lu",
      "mc",
      "mo",
      "nl",
      "no",
      "nz",
      "pl",
      "pt",
      "se",
      "sg",
      "si",
      "sk",
      "tw"
```